src/BlazorWebView/src/core/Windows/IWebView2Wrapper.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Web.WebView2.Core;
+
+namespace Microsoft.AspNetCore.Components.WebView.WebView2
+{
+	/// <summary>
+	/// Provides an abstraction for different UI frameworks to provide access to APIs from
+	/// <see cref="Microsoft.Web.WebView2.Core.CoreWebView2"/> and related controls.
+	/// </summary>
+	public interface IWebView2Wrapper
+	{
+		/// <summary>
+		/// Gets the <see cref="CoreWebView2"/> instance on the control. This is only available
+		/// once the <see cref="Task"/> returned by <see cref="EnsureCoreWebView2Async(CoreWebView2Environment)"/>
+		/// has completed.
+		/// </summary>
+		CoreWebView2 CoreWebView2 { get; }
+
+		/// <summary>
+		/// Gets or sets the source URI of the control. Setting the source URI causes page navigation.
+		/// </summary>
+		Uri Source { get; set; }
+
+		/// <summary>
+		/// Initializes the <see cref="CoreWebView2"/> instance on the control. This should only be called once
+		/// per control.
+		/// </summary>
+		/// <param name="environment">A <see cref="CoreWebView2Environment"/> that can be used to customize the control's behavior.</param>
+		/// <returns>A <see cref="Task"/> that will complete once the <see cref="CoreWebView2"/> is initialized and attached to the control.</returns>
+		Task EnsureCoreWebView2Async(CoreWebView2Environment? environment = null);
+
+		/// <summary>
+		/// Event that occurs when an accelerator key is pressed.
+		/// </summary>
+		event EventHandler<CoreWebView2AcceleratorKeyPressedEventArgs> AcceleratorKeyPressed;
```

Comment:
This seems incredibly specific, what's the reasoning behind it?

Reply:
Ask @SteveSandersonMS 😄 It's here because it's from the Blazor WebView2 abstractions so we could abstract between WPF/WinFOrms, because this event is on the _control_ and not on the common WebView2Core type. Ironically it's not even used here because this API doesn't exist in WinUI! So it's here in order for the file to remain identical, even though it isn't used.